cassandra/cassandra_main.py

```diff
@@ -90,21 +82,56 @@ def bootstrap_sp(cfgfile_name):
     for thread in component_threads:
         thread.join()
 
-    # If this is a multiprocessing calculation, then we need to
-    # perform the finalization procedure
-    if argvals.mp:
-        finalize(component_list[0], threads[0])
-
     # Check to see if any of the components failed
     fail = 0
     for component in component_list:
         if component.status != 1:
-            print(f'Component {str(component.__class__)} returned failure status\n')
+            from logging import error
```

Comment:
As you can see here, and in several other places, we're sort of slouching toward implementing #30.  It's not technically on the work plan, but we could score a quick hit by replacing the rest of the `print`s with logging calls, and maybe rolling in a solution for #44 along the way.